rfcs/0011_Arm_Ethos-U_Integration.md

```diff
@@ -192,42 +191,42 @@ TVM_DLL int32_t ethosu_0(TVMValue* args, int* type_code, int num_args, TVMValue*
 
 ## Build system
 
-The only dependency of TVM compilation for Ethos™-U is using Arm® Vela compiler (https://pypi.org/project/ethos-u-vela/).
-However, to run inferences with the sources generated by TVM, we would need to use the Ethos-U core driver (https://git.mlplatform.org/ml/ethos-u/ethos-u-core-driver.git/about/). The user is expected to include the necessary sources if they were looking to use this bare-metal.
+The only dependency of TVM compilation for Ethos™-U NPU is using [Arm® Vela compiler](https://pypi.org/project/ethos-u-vela/).
+However, to run inferences with the sources generated by TVM, we would need to use the [Ethos™-U NPU core driver](https://git.mlplatform.org/ml/ethos-u/ethos-u-core-driver.git/about/). The user is expected to include the necessary sources if they were looking to use this bare-metal.
 
 ## Testing
 
 Firstly, we will be providing unit tests for the components described above.
 
-Secondly, we are planning to use Arm® Corestone™-300 Fixed Virtual Platform  (FVP – https://developer.arm.com/ip-products/subsystem/corstone/corstone-300) in the CI to be able to simulate the codegen'd artifacts of TVM on a SoC that has Arm® Cortex™-M55 and Ethos™-U55.
+Secondly, we are planning to use [Arm® Corstone™-300 reference system](https://developer.arm.com/ip-products/subsystem/corstone/corstone-300) in the CI to be able to simulate the codegen'd artifacts of TVM on a SoC that has Arm® Cortex™-M55 and Ethos™-U55.
```

Comment:
Will there ever be plans to provide third party CI for testing on a hardware cloud?

Reply:
@leandron

Reply:
cc : @u99127, However I feel this question is out of scope for the RFC.

Reply:
Thanks @manupa-arm 

We currently do not have any plans for adding hardware into a cloud. Ethos-U is licensed as a part of Arm’s IP portfolio.  I expect SoCs and boards containing Ethos-U from the broader Arm ecosystem in due course but I’m not at liberty to speculate on time lines.

As part of the Ethos-U upstreaming work and the first PR that has been merged for docker images, Arm has contributed a reference system, what we call Fixed Virtual Platforms for the Corstone-300 subsystem in the TVM CI.  We believe this is sufficient for continually testing the correctness of the Ethos-U port and indeed for testing correctness on Cortex-M.

The value of hardware based correctness testing is debatable beyond testing that comes from the simulator and is beyond the scope of this RFC or Pull Requests.